includes/admin/views/status/view-log.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Log file viewer
+ *
+ * Used on the LifterLMS Admin Status Logs tab to output a single log file.
+ *
+ * @package LifterLMS/Admin/Views
+ *
+ * @since [version]
+ * @version [version]
+ *
+ * @property array[] $logs    Associative array of log files. The array key is the file "slug" and the value is the file's absolute path.
+ * @property string  $current Slug of the current log file.
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+// Nonce URL to delete a log file.
+$delete_url = 'debug-log' === $current ? '' : wp_nonce_url(
+	add_query_arg(
+		array(
+			'llms_delete_log' => $current,
+		),
+		admin_url( 'admin.php?page=llms-status&tab=logs' )
+	),
+	'delete_log'
+);
```

Comment:
1) I think this part can be moved into includes/admin/class.llms.admin.page.status.php 
2) the condition here is a little bit weak. The "wp" debug log file can be named in other ways, that's why we get its actual name via `ini_get()`. Right? So I guess that here we should get the current file name (`$logs[$current]`) and compare it with `ini_get( 'error_log' )`. What do you think?

Reply:
1. I guess lol
2. This is referencing the "key" that is given to the log in the array of available logs which is hardcoded as "debug-log" in the `get_logs()` method so regardless of what the file is actually named the "slug" we give to that file will always be "debug-log" so this condition will always work.

Reply:
Fixed 1

Reply:
1. I'm not using the 'I guess' the right way, I guess :)
2. Oh mmm sorry, totally stupid me :(